src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -163,13 +163,15 @@ class primitive_inst {
     virtual void update_weights() {}
     void update_impl();
 
+    std::string get_layout_key();
```

Comment:
I think this is not needed in the primitive_inst class but enough to be used as a lambda function

Reply:
Fixed it